test/data/gtfs_test/frequencies.txt

```diff
@@ -0,0 +1,2 @@
+trip_id,start_time,end_time,headway_secs,exact_times
+bar,00:00:00,02:00:00,1800,0
```

Comment:
should also probably have an example where its schedule based (ie exact_times == 1)